docs/reports/unmapped_omim.md

```diff
@@ -4,6 +4,19 @@
 ### Unmapped mappable terms _(!excluded, !deprecated)_
```

Comment:
Labels are now appearing:

| subject_id    | subject_label                                                                                                      |
|:--------------|:-------------------------------------------------------------------------------------------------------------------|
| OMIMPS:109400 | Basal cell nevus syndrome                                                                                          |
| OMIMPS:613652 | C1q deficiency                                                                                                     |
| OMIMPS:301108 | Cataracts, hearing impairment, nephrotic syndrome, and enterocolitis 1                                             |
| OMIMPS:164210 | Craniofacial Microsomia                                                                                            |
| OMIMPS:113800 | Epidermolytic hyperkeratosis                                                                                       |
| OMIMPS:605899 | Glycine encephalopathy                                                                                             |
| OMIMPS:141200 | Hematuria, benign familial                                                                                         |
| OMIMPS:603896 | Leukoencephalopathy with vanishing white matter                                                                    |
| OMIMPS:278000 | Lysosomal acid lipase deficiency                                                                                   |
| OMIMPS:609015 | Mitochondrial trifunctional protein deficiency                                                                     |
| OMIMPS:167030 | Nephrolithiasis, calcium oxalate                                                                                   |
| OMIMPS:608415 | Prolonged electroretinal response suppression                                                                      |
| OMIMPS:177735 | Pseudohypoaldosteronism, type I                                                                                    |

Reply:
@matentzn This is now ready for review.